enhancements/network/user-defined-network-segmentation.md

```diff
@@ -357,11 +359,106 @@ block/allow cluster egress or ingress traffic will still be enforced.
+| Topology       | The topological configuration for the network. Must be one of `layer2` or `layer3`.                                                                                                                                                                                                                                                                                                                | No       |
+| Role           | Select the network role in the pod, either `primary` or `secondary`.                                                                                                                                                                                                                                                                                                                               | No       |
+| MTU            | The maximum transmission unit (MTU).<br/>The default value, 1300, is automatically set by the kernel                                                                                                                                                                                                                                                                                               | Yes      |
+| Subnets        | The subnet to use for the network across the cluster.<br/>E.g. 10.100.200.0/24.<br/>IPv6 (2001:DBB::/64) and dual-stack (192.168.100.0/24,2001:DBB::/64) subnets are supported.<br/>When omitted, the logical switch implementing the network only provides layer 2 communication, and users must configure IP addresses.<br/>Port security only prevents MAC spoofing if the subnets are omitted. | Yes      |
+| ExcludeSubnets | List of CIDRs and IP addresses.<br/>IP addresses are removed from the assignable IP address pool and are never passed to the pods.                                                                                                                                                                                                                                                                 | Yes      |
+
+The cluster scoped CRD should have the following additional field:
+
+| Field name | Description                                                                                           | optional |
+|------------|-------------------------------------------------------------------------------------------------------|----------|
+| Selector   | Key-value list of labels used as a selector for which namespace the network should be available for.  | No       |
+
+Since `localnet` topology for primary networks is non-goal, the CRDs controller should reject such requests.
+
+##### status
+The CRD status should reflect the NAD state through conditions.
+For example, when the NAD its created the condition should be placed with status `True`.
```

Comment:
Should we say when the NAD is created and OVN-Kubernetes has programmed this network into OVN, OVN-Kubernetes will update the status?

Reply:
> Since the suggested solution utilize the CNI model where the underlying ovn network is set by ovn-k CNI plugin when pod is created, and deleted when pod is deleted. With the suggested solution, the status is actually updated when the NAD is created, 

so what does a "true" status indicate? and when the status be false?

> the underlying ovn network is created later on, once the consumer pod is created.

Reply:
I think he means NetworkCreated == the network topology has been created in OVN. NetworkReady== the NAD has been created.

I think this is a good idea.

Reply:
Following offline discussion, it was concluded condition should reflect underlying OVN network state, reflecting the NAD state is not sufficient.

Thanks for the discussion and explaining :slightly_smiling_face: 
Added text to describe status should reflect underlying OVN network state, and updated examples.